src/main/java/com/impactupgrade/nucleus/client/MinistryByTextClient.java

```diff
@@ -190,4 +232,77 @@ public String toString() {
           '}';
     }
   }
+
+  @JsonIgnoreProperties(ignoreUnknown = true)
+  public static class Contact {
+    public String firstName;
+    public String lastName;
+    public String gender;
+    public String mobileNo;
+    public SubscriberAddress address;
+    public String region;
+  }
+
+  @JsonIgnoreProperties(ignoreUnknown = true)
+  public static class BulkOperationResponse {
+    public String message;
+    public boolean isError;
+    public String appCode;
+    public Data data;
+
+    @Override
+    public String toString() {
+      return "BulkOperationResponse{" +
+              "message='" + message + '\'' +
+              ", isError=" + isError +
+              ", appCode='" + appCode + '\'' +
+              ", data=" + data +
+              '}';
+    }
+  }
+
+  public static class Data {
+    public String batchId;
+
+    @Override
+    public String toString() {
+      return "Data{" +
+              "batchId='" + batchId + '\'' +
+              '}';
+    }
+  }
+
+  public static void main(String[] args) {
+    Environment env = new Environment() {
+      @Override
+      public EnvironmentConfig getConfig() {
+
+        EnvironmentConfig.MBT mbt = new EnvironmentConfig.MBT();
+        mbt.clientId = "GVU05RE7VMACNUU96ME8";
+        mbt.clientSecret = "CPMw462MQdLTW6MDDlagUWlOhxXXJgDRc0D8cBHuUqhO=g6ELo";
+        mbt.campusId = "cf774a3b-4910-4b16-b6b0-608f80d216a4";
```

Comment:
Reminder: nucleus-engine is fully open source and public now, so we need to be careful not to expose client secrets. But, assuming this is for the sandbox, which is fine. Remove the whole main method?